HW1/P2/P2b.py

```diff
@@ -1,3 +1,16 @@
 from P2 import *
+from random import *
 
-# Your code here
+# Draw the figure
+def f_random(key):
+    return randrange(1,101,1)
+    
+xy = [(i,j) for j in range(1,2001) for i in range(1,2001)]
+arr = sc.parallelize(xy, 1).partitionBy(numPartitions=100, partitionFunc=f_random)
```

Comment:
Load partitions using random assignment. In expectation, hopes for load balanced solution.